daemon/api_validate_test.go

```diff
@@ -630,6 +630,15 @@ func (s *apiValidationSetsSuite) TestApplyValidationSetMonitorModePinnedLocalOnl
 
 	rsp := s.syncReq(c, req, nil)
 	c.Assert(rsp.Status, check.Equals, 200)
+	res := rsp.Result.(daemon.ValidationSetResult)
+	c.Check(res, check.DeepEquals, daemon.ValidationSetResult{
+		AccountID: s.dev1acct.AccountID(),
+		Name:      "bar",
+		Mode:      "monitor",
+		PinnedAt:  99,
+		Sequence:  99999,
+		Valid:     true,
```

Comment:
I would have expected more tests, at least an enforce test to also check this?

Reply:
Right, added.